publish-nuget/action.yml

```diff
@@ -36,20 +36,29 @@ runs:
       id: validate-publish-type
       shell: bash
       run: |
+        PUBLISH_MODE="${{ inputs.publishMode }}"
```

Comment:
If we manually dispatch the `nuget-yml` workflow, it means that we need to publish all packages, so it will ignore the `publishMode` input sent and override with `always`.